src/PowerShellEditorServices/Session/Host/EditorServicesPSHost.cs

```diff
@@ -78,14 +79,165 @@ public override string Name
         }
 
         /// <summary>
-        ///  
+        ///
+        /// </summary>
+        public class ConsoleColorProxy
+        {
+            private EditorServicesPSHostUserInterface _hostUserInterface;
+
+            /// <summary>
+            ///
+            /// </summary>
+            public ConsoleColorProxy(EditorServicesPSHostUserInterface hostUserInterface)
+            {
+                if (hostUserInterface == null) throw new ArgumentNullException("hostUserInterface");
+                _hostUserInterface = hostUserInterface;
+            }
+
+            /// <summary>
+            ///
+            /// </summary>
+            public ConsoleColor ErrorForegroundColor
+            {
+                [SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode")]
```

Comment:
All of the diagnostic suppressions (`SuppressMessage`) can be removed since we don't have the same analyzers on the project

Reply:
@SeeminglyScience are you sure ? Because AppVeyor is stuck since 15 minutes with this push.